plugins/module_utils/repo_setup/main.py

```diff
@@ -577,7 +530,6 @@ def install_deps(args, base_path):
             content = _change_priority(content, 20)
             _write_repo(content, args.output_path, name="delorean-current-podified")
             content = _get_repo(base_path + "current/delorean.repo", args)
```

Comment:
Why current-podified-dev is needed at all now since include pkgs are not used? without it always current repos will be used for all the openstack rpms as that will be higher priority.